packages/shortcut/index.ts

```diff
@@ -32,7 +32,7 @@ interface ComposeMethod {
 }
 
 // TODO: enhance typing
-export type ComposeProperty = ComposeMethod & Descriptor;
+export type ComposeProperty = ComposeMethod & Descriptor & PropertyMap;
```

Comment:
The `.compose` property is both a method and a descriptor. But not a random key-value map. Please, remove `PropertyMap` from here too.
Sorry, I noticed too late. I'm on a vacation atm. :)

Reply:
Will do